runtime/gc_base/CMakeLists.txt

```diff
@@ -62,3 +62,15 @@ target_link_libraries(j9gcbase
 		omrgc
 )
 
+target_enable_ddr(j9gcbase j9ddr)
+set_property(TARGET j9gcbase PROPERTY DDR_HEADERS
```

Comment:
Why does this only add specific header files, rather than all like `shared_common`?

Reply:
> The current process also considers jvmti, port, j9vm and jcl: why are they not included here?

They are already enabled. See #5591